src/client/components/va-benefits/VAInformationSection.jsx

```diff
@@ -35,31 +35,22 @@ class VAInformationSection extends React.Component {
         );
     } else {
       content = (<fieldset>
-        {/* TODO: Change the headings to something related to the questions. */}
-        <legend>Veteran</legend>
-        <p>
-          Please review the following list and select all the responses that apply to you.
-          This information will be used to determine which sections of the Application for
-          Health Benefits you should complete.
-        </p>
+        <legend>VA Benefits</legend>
         <p>(<span className="hca-required-span">*</span>) Indicates a required field</p>
         <div className="input-section">
           <ErrorableRadioButtons required
-              errorMessage={validateIfDirty(this.props.data.isVaServiceConnected, isNotBlank) ? '' : 'Please select a response'}
-              label="Are you VA Service Connected 50% to 100% Disabled?"
-              options={yesNo}
-              value={this.props.data.isVaServiceConnected}
-              onValueChange={(update) => {this.props.onStateChange('isVaServiceConnected', update);}}/>
-
-          <ErrorableRadioButtons required
               errorMessage={validateIfDirty(this.props.data.compensableVaServiceConnected, isNotBlank) ? '' : 'Please select a response'}
-              label="Are you compensable VA Service Connected 0% - 40%?"
+              label="Are you currently receiving monetary compensation for a service-connected disability with a rating of 0% - 40% disabled?"
               options={yesNo}
               value={this.props.data.compensableVaServiceConnected}
               onValueChange={(update) => {this.props.onStateChange('compensableVaServiceConnected', update);}}/>
-          <span>
-            A VA determination that a Service-connected disability is severe enough to warrant monetary compensation.
-          </span>
+
+          <ErrorableRadioButtons required
+              errorMessage={validateIfDirty(this.props.data.isVaServiceConnected, isNotBlank) ? '' : 'Please select a response'}
+              label="Are you currently receiving monetary compensation for a service-connected disability with a rating of 50% to 100% disabled?"
```

Comment:
Yeah, it's not great. The reason I kept it this way is because I was encountering some problems doing it the ways we talked about yesterday: 
1. I couldn't bold just the percentages because the whole label is being passed as a prop on ErrorableRadioButtons, so I didn't know how to bold just part of it. 
2. I was worried about accessibility issues if I extracted the main content of the label and had the labels simply be the percentages. Would that make sense to someone using a screen reader? I'm guessing not. 

If you have suggestions for how to handle this differently I'm all ears! Unfortunately this was the best I could come up with for now.